src/components/Audios.vue

```diff
@@ -114,42 +114,31 @@ export default {
 
 <style scoped lang="scss">
 audio {
-	background-color: black;
+	/* over arrows in tiny screens */
+	z-index: 20050;
```

Comment:
Are you sure about this? Would this completely disable switching to the next/previous video/audio on mobile as swiping is also disable IIRC?

Reply:
You also may remove the media query that is done at the bottom of the file in this case...

Reply:
I just reordered the properties, it was already here :grin: 
![image](https://user-images.githubusercontent.com/14975046/186089312-a16a88c2-6d7e-4e96-abf4-2acf43f42d85.png)